lib/vdsm/virt/thinp.py

```diff
@@ -343,20 +343,15 @@ def _should_extend_volume(self, drive, volumeID, block_info):
             # next lvm extent.
```

Comment:
"querying block status may that the" -> "querying block status may state that the"